### DISCUSSION TITLE
Is multiple files sync/batch sync something planned?

### DISCUSSION THREAD

**Discussion by chs33** [windingwind/zotero-better-notes#785] [Q&A] [answered]
Right now seems like the sync button in sync manager and auto sync is only for workspace note, how feasible/is there any plan for multiple files sync/batch sync?  Maybe auto sync for large number of notes is too heavy, I am thinking about in sync manager where you can select all/select a ranges of notes based on some time stamp to conduct manual sync in batch.

**Comment by windingwind** [comment, marked as answer]
To batch sync:

<img width="337" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/40b7dede-c4e9-4f45-aced-9c3911b880b4">

**Comment by chs33** [comment]
But if those notes are already being tracked in sync manager, you can not  batch sync them in sync manager right?(technically you can still do what you suggest above though) I am more talking about the case where you make change on the multiple files on markdown side and want to sync back to Zotero(and those markdowns have already previously synced with notes)

**Reply by windingwind** [reply]
actually, you can.

<img width="800" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/470af3bd-fa1e-43c8-a1a4-d2d67a2f8394">

And if you have auto-sync enabled, BN will periodically check the status of note and markdown in the background.

**Reply by chs33** [reply]
strangely it doesn't work for me... 
![image](https://github.com/windingwind/zotero-better-notes/assets/16418270/8ffed356-2afc-4d61-9eb5-3b7c021cc75e)

**Reply by windingwind** [reply]
doesn't it sync all selected notes, if you select all and press `sync`?

If some note items are opened, they'll get skipped during auto-syncing.

**Reply by chs33** [reply]
previously it doesn't sync anything. I manually unsync and sync again it does the work now. Thank you.

**Reply by windingwind** [reply]
I believe that's a bug. Please start an issue with debug outputs (see steps in the issue template) that records what happens after you press `sync`.
